README.rst

```diff
@@ -194,6 +194,20 @@ Content:
 .. code-block:: python3
 
    {
-     'reason': 'link_enabled'
+     'reason': 'link enabled'
      'link': <Link object>
    }
+
+
+kytos/topology.link.deleted
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Event reporting that a link was deleted. It contains the link instance.
+
+Content:
+
+.. code-block:: python3
+  {
+    'link': <Link object>
+    'reason': 'link deleted'
```

Comment:
@Alopalao the `reason` key value on `kytos/topology.link.deleted` event I think ended up a bit redundant, on `kytos/topology.link_down` we have it because multiple reasons can lead to a link down, but deletion will only have a single one, so unless you're seeing any other reason to keep a reason here (let me know if you do) I'd say let's nuke the `reason` part.